ultravox/model/ultravox_model.py

```diff
@@ -33,6 +33,8 @@ class UltravoxModel(transformers.LlamaPreTrainedModel):
     config_class = UltravoxConfig
     config: UltravoxConfig  # for type hinting
     _no_split_modules = ["Wav2Vec2Model", "WhisperEncoder", "LlamaDecoderLayer"]
+    _keys_to_ignore_on_load_missing = ["audio_tower.*"]
```

Comment:
This one is technically not required, but I think having it will not hurt.

Reply:
please add a comment explaining what these lines do, for posterity

Reply:
Added a three line explanation for each :)